src/factories/ModuleFactory.sol

```diff
@@ -1,9 +1,11 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 pragma solidity 0.8.19;
 
-// External Dependencies
-import {Context} from "@oz/utils/Context.sol";
+//External Dependencies
+import {ERC2771Context} from "@oz/metatx/ERC2771Context.sol";
 import {Ownable2Step} from "@oz/access/Ownable2Step.sol";
+import {Ownable} from "@oz/access/Ownable.sol";
+import {Context} from "@oz/utils/Context.sol";
```

Comment:
Because the Factory is inheriting from Ownable2Step and ERC2771Context on two seperate occasions I have to explicitly override them for the implementation of the ERC2771. For that I need the reference.
In the fix I shifted it so that it reads the import from Ownable2Step specifically maybe that makes it more readable.